src/steps/account/account-field-equals.ts

```diff
@@ -60,15 +62,22 @@ export class AccountFieldEquals extends BaseStep implements StepInterface {
       } else if (!account[0].hasOwnProperty(stepData.field)) {
         // If the given field does not exist on the account, return an error.
         return this.error('The %s field does not exist on Account %s', [field, identifier]);
-      } else if (this.compare(operator, account[0][field], expectedValue)) {
+      } else {
+        actualValue = account[0][field];
```

Comment:
@saludangelito 🤔 I can see how this helps with the display in the error message, but if we were to take this approach, we'd have to duplicate the logic in check steps for each object (e.g. lead, opportunity, etc), and remember to apply the same logic in future object check steps.

What if instead, we kept the `JSON.stringify()` in the `BaseStep::compare()` method, but did an additional check for `isObject()` in the `BaseStep::outcomelessResponse()` method: https://github.com/run-crank/cog-salesforce/blob/master/src/core/base-step.ts#L94

Reply:
That.... is a great idea! The other objects honestly slipped my mind since I've been moving from cog to cog, sorry about that.